src/test/resources/3wt.svg

```diff
@@ -46,7 +46,17 @@
   40% {stroke: #00BCD4; stroke-width: 0.15}
 }
 ]]></style>
-    <metadata></metadata>
+    <metadata>
+        <nodes>
+            <id diagram="0" equipment="VL_132"/>
```

Comment:
I'm wondering if we shouldn't enrich later the data for each node. Then it would be more appropriate to have the `node` tag instead of `id` tag.